CONTRIBUTING.md

```diff
@@ -31,88 +30,57 @@ By participating, you are expected to uphold this code. Please report unacceptab
 
 * [EthSigner documentation]
 * [Gitter]
 
 ## How To Contribute
-### Reporting Bugs
+### Reporting Documentation Bugs
 
-This section guides you through submitting a bug report. Following these guidelines helps maintainers 
-and the community understand your report, reproduce the behavior, and find related reports.
+This section guides you through submitting a bug report for documentation. Following these guidelines helps maintainers 
+and the community understand your report and find related reports.
 
 Before creating bug reports, please check the [before-submitting-a-bug-report](#before-submitting-a-bug-report) 
-checklist as you might find out that you don't need to create one. When you are creating a bug report, 
-please [include as many details as possible](#how-do-i-submit-a-good-bug-report).
+checklist as you might find out that you don't need to create one. When you create a bug report, [include as many details as possible](#how-do-i-submit-a-good-bug-report).
 
-> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, 
+> **Note:** If you find a **Closed** issue that seems to address the same problem that you're experiencing, 
 open a new issue and include a link to the original issue in the body of your new one.
 
 #### Before Submitting A Bug Report
-* **Confirm the problem** is reproducible in the latest version of the software
-* **Check the [Debugging documentation]**. You might be able to find the cause of the problem and fix things yourself. 
-* **Perform a [cursory search of project issues](https://pegasys1.atlassian.net/browse/PAN-2502?jql=project%20%3D%20PAN)** 
+* **Perform a [cursory search of project issues](https://pegasys1.atlassian.net/browse/ES-27?jql=project%20%3D%20ES)** 
 to see if the problem has already been reported. If it has **and the issue is still open**, add a comment 
 to the existing issue instead of opening a new one.
 
 #### How Do I Submit A (Good) Bug Report?
 Bugs are tracked as [Jira issues](https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10000).  
 
-Explain the problem and include additional details to help maintainers reproduce the problem:
+Explain the problem and include additional details to help maintainers understand the problem:
 
 * **Use a clear and descriptive summary** for the issue to identify the problem.
-* **Describe the exact steps which reproduce the problem** in as many details as possible. For example, 
-start by explaining how you started EthSigner, e.g. which command exactly you used in the terminal, 
-or how you started it otherwise. 
+* **Describe the problem** in as many details as possible. If screenshots will help to explain the problem,
+attach them in the report. 
 * **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, 
-or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, 
```

Comment:
Having these context element sin the bug description would still be useful. COuld we keep some of them and just translate "the software" to "the documentation"?

Reply:
Re-added some of this information.